kahuna/public/js/components/gr-add-label/gr-add-label.js

```diff
@@ -58,7 +61,8 @@ addLabel.directive('grAddLabel', [function () {
         scope: {
             image: '=',
             grSmall: '=?',
-            active: '='
+            active: '=',
+            selected: '='
```

Comment:
Could we rename `selected` to `images`? Otherwise the concept of selection is leaking into this generic component, which is unnecessary and confusing.

(We could even reduce the API surface and just have `images`, and when needed with a single image, it could just be passed `images="[ctrl.theImage]"`, but I don't mind too much.)